eZ/Bundle/EzPublishCoreBundle/Command/ReindexCommand.php

```diff
@@ -209,6 +210,21 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             );
             $this->searchIndexer->createSearchIndex($output, (int) $iterationCount, !$commit);
         } else {
+            $io = new SymfonyStyle($input, $output);
+            $io->warning(<<<EOT
+For optimal performance, before running this command, make sure that:
+- the xdebug extension is disabled,
+- you're running the command in "prod" environment,
+- memory limit for big databases is set to "-1" or an adequately high value,
+- --iteration-count is low enough (default: 50),
+- number of processes for parallel batch operations is high enough (default: 'auto' is a good choice).
+EOT
+            );
```

Comment:
PHP 7.3 allows indentation in HEREDOC notation:
```suggestion
            $io->warning(<<<EOT
                For optimal performance, before running this command, make sure that:
                - the xdebug extension is disabled,
                - you're running the command in "prod" environment,
                - memory limit for big databases is set to "-1" or an adequately high value,
                - --iteration-count is low enough (default: 50),
                - number of processes for parallel batch operations is high enough (default: 'auto' is a good choice).
            EOT);
```
but Andre's comment about making the command more robust by checking those things (Is extension enabled? What's the memory limit?) still stands.